frontend/src/lib/k8s/apiProxy.ts

```diff
@@ -68,6 +68,13 @@ export async function request(
     clearTimeout(id);
   }
 
+  // The backend signals through this CORS safe header that it wants a reload.
+  // See pluginwatcher.go
+  const headerVal = response.headers.get('Content-Language');
```

Comment:
In https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS I see this:
```WebKit Nightly and Safari Technology Preview place additional restrictions on the values allowed in the Accept, Accept-Language, and Content-Language headers. If any of those headers have ”nonstandard” values, WebKit/Safari does not consider the request to be a “simple request”. What values WebKit/Safari consider “nonstandard” is not documented, except in the following WebKit bugs:
```

AFAIU, shouldn't be a problem except for users who use the headless mode (if we restrict the plugin watch to desktop use) with a non-final version of Safari, but I wonder if we'll hit an issue in the future.

Doing a quick search, have you tried adding a dedicated allowed header?
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Expose-Headers

Reply:
I used Access-Control-Expose-Headers in the end, which works easily in dev mode and with the app.